src/solver/conv_asm_implicit_gemm_gtc_bwd.cpp

```diff
@@ -230,7 +226,7 @@ static bool FindImplicitGemmGtcDynamicBwdKernel(const ConvolutionContext& ctx,
             bool is_gemm_not_empty = gemm_k > 0;
             if(is_gemm_not_empty)
             {
-                if(gemm_k % pConfig->gemm_k_per_block != 0)
+                if((pConfig->gemm_k_per_block == 0) || (gemm_k % pConfig->gemm_k_per_block != 0))
```

Comment:
Ditto

Reply:
same as above, pls check the latest commit.